util/cron/nightlysubs.pm

```diff
@@ -117,7 +117,11 @@ sub endMailHeader {
 }
 
 sub endMailChplenv {
-    my $chplenv = `$chplhomedir/util/printchplenv --debug`;
+    my $ch = $chplhomedir;
```

Comment:
$chplhomedir is used throughout nightly, which uses this perl module. In most cases, we want it to use the relative path. I think printchplenv and calls to chplenv/ scripts are the only cases where we want to use the real CHPL_HOME instead of a relative one (if there is a difference).

Note that nightly testing on the cray hw doesn't use nightly, so I haven't made an attempt to clean up other calls to printchplenv or chplenv/ scripts there.